django/db/backends/sqlite3/features.py

```diff
@@ -49,6 +49,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         'cs': 'binary',
         'non_default': 'nocase',
     }
+    supports_update_conflicts_with_target = Database.sqlite_version_info >= (3, 24, 0)
+    supports_update_conflicts = True
```

Comment:
Please flip these around:

```suggestion
    supports_update_conflicts = True
    supports_update_conflicts_with_target = Database.sqlite_version_info >= (3, 24, 0)
```